openpype/tools/workfiles/lib.py

```diff
@@ -0,0 +1,214 @@
+import os
+import shutil
+import uuid
+import time
+import json
+import logging
+import contextlib
+
+import appdirs
+
+
+class TempPublishFilesItem(object):
+    """Object representing on subfolder in app temp files.
```

Comment:
Seems like a typo? Not sure what is being referred to with "Object representing on subfolder"?